### ISSUE TITLE
Error running the example

### ISSUE THREAD

**Issue by jotyGill** [alexbotello/python-overwatch#2]
Running the example with python 3.6.4 on Fedora, errors out.
```bash
from overwatch import Overwatch

overwatch = Overwatch(battletag="Okush#11324")
print(overwatch.playtime)
```

`Traceback (most recent call last):
  File "test.py", line 4, in <module>
    print(overwatch.playtime)
  File "/usr/local/lib/python3.6/site-packages/overwatch/__init__.py", line 79, in playtime
    return self._generate_comparisons(compare['playtime']);
  File "/usr/local/lib/python3.6/site-packages/overwatch/__init__.py", line 74, in _generate_comparisons
    comparison = data[self.mode]
AttributeError: 'Overwatch' object has no attribute 'mode'
`

**Comment by jotyGill** [comment]
issue is with "overwatch.playtime"

overwatch() for example, works fine.

**Comment by alexbotello** [comment]
Thanks for the report. I’ll have a bug fix out for this later today.

**Comment by alexbotello** [comment]
Issue is fixed with `1.0.6`
